cmd/plugin/builder/template/plugintemplates/dotgolangci.txt.tmpl

```diff
@@ -1,4 +1,9 @@
 linters-settings:
```

Comment:
I used the same settings as we use in the tanzu-cli project, after removing the two that were not applicable.